backend/cache_utils.py

```
@@ -86,3 +86,30 @@ def invalidate_tab_feeds_cache(tab_id, invalidate_tabs=True):
+        invalidate_tabs (bool): If True, also invalidates the main tabs list cache.
+    """
+    # Prevent generator exhaustion by casting to a list
+    tab_ids_list = list(tab_ids)
+    if not tab_ids_list:
+        return
+
+    version_keys = [get_tab_version_key(tid) for tid in tab_ids_list]
+    current_versions = cache.get_many(*version_keys)
+
+    new_versions_dict = {}
+    for key, current_val in zip(version_keys, current_versions):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)